website/docs/introduction/git-cheat-sheet.md

```diff
@@ -76,7 +76,7 @@ See `sl help revset` for more ways of referencing commits.
 |Commit changes |git commit -a |sl commit |
 |Modify commit |git commit -a --amend |sl amend |
 |Move to commit |git checkout COMMIT |sl goto COMMIT |
-|Remove a commit |git reset -hard COMMIT |sl hide COMMIT |
+|Remove current commit |git reset --hard HEAD^ |sl hide . |
```

Comment:
I think it's important to include the more general "Remove a commit" use case, though.

Reply:
@bolinfest What do you propose for the more general "remove a commit" case in Git? There isn't really an easy way to do that. I suppose you could try to use `git rebase -i` from all branches for which the commit is reachable and manually remove it, or, if the commit is the tip of some branch(es), you can use `git branch -f`/`git reset --hard` (depending on whether or not the branch is checked out 🙄). As the example is previously written, it wasn't removing a commit in the general case, anyways.

The Sapling approach to removing a commit isn't fully general, either: in my experimentation, it will also hide all descendant commits, so you can't remove just one commit in the middle of a stack.

Reply:
I think the current way is fine. Removing an arbitrary commit might require knowledge about the graph topology, which seems hard to explain in a small table cell.